orchagent/main.cpp

```diff
@@ -543,6 +544,29 @@ int main(int argc, char **argv)
     attr.value.u64 = gSwitchId;
     attrs.push_back(attr);
 
+
+    char *platform = getenv("platform");
+    if (platform && strstr(platform, BRCM_PLATFORM_SUBSTRING))
```

Comment:
can this be done for switchtype fabric?

Reply:
This change/the code path is applicable for both fabric and voq in chassis. In both "fabric" (supervisor card) and in "voq" (asic in linecard) the switch create with bcm SAI takes more than the default time.